HW2

```diff
@@ -0,0 +1,30 @@
+#1. Historic Batting Average
+
+CREATE OR REPLACE TABLE HIS AS SELECT bc.batter as Batter, SUM(bc.Hit) / SUM(bc.atBat) as Average
+FROM batter_counts bc WHERE bc.atBat <> 0 group by bc.batter;
+
+
+#2 Annual Batting Average
+
+CREATE TABLE ANNUAL as SELECT YEAR(g.local_date), bc.batter, SUM(bc.Hit) / SUM(bc.atBat) 
+AS Average FROM batter_counts bc JOIN game g ON bc.game_id = g.game_id
+WHERE bc.atBat <> 0
```

Comment:
I see that you're using atBat <> 0 to avoid the Division By 0 error. But then you're leaving out those rows that have atBat = 0, in which case I think we can use NULLIF() function to turn them to NULL values and thus will yield NULL values in the Average column.

Reply:
Updated!